qsimcirq_tests/qsimcirq_test.py

```diff
@@ -215,6 +215,39 @@ def test_invalid_params():
     _ = qsim_simulator.simulate_sweep(circuit, params=prs)
 
 
+def test_iterable_qubit_order():
+  # Check to confirm that iterable qubit_order works in all cases.
+  q0, q1 = cirq.LineQubit.range(2)
+  circuit = cirq.Circuit(
+    cirq.H(q0), cirq.H(q1),
+  )
+  qsim_simulator = qsimcirq.QSimSimulator()
+
+  assert qsim_simulator.compute_amplitudes(
+    circuit,
+    bitstrings=[0b00, 0b01],
+    qubit_order=reversed([q1, q0]),
+  ) == qsim_simulator.compute_amplitudes(circuit, bitstrings=[0b00, 0b01])
+
+  assert (
+    qsim_simulator.simulate(circuit, qubit_order=reversed([q1, q0])) ==
+    qsim_simulator.simulate(circuit, qubit_order=reversed([q1, q0]))
```

Comment:
Should that be just `qsim_simulator.simulate(circuit)` in line 234?

Reply:
Yup, thanks for the catch!